src/Core/src/Platform/iOS/TransformationExtensions.cs

```diff
@@ -0,0 +1,159 @@
+﻿using System;
+using CoreAnimation;
+using CoreGraphics;
+using Microsoft.Maui.Graphics;
+using UIKit;
+
+namespace Microsoft.Maui
+{
+	public static class TransformationExtensions
+	{
+		public static void UpdateTransformation(this UIView nativeView, IView? view)
+		{
+			CALayer? layer = nativeView.Layer;
+			bool isInteractive = nativeView.UserInteractionEnabled;
+			CGPoint? originalAnchor = layer?.AnchorPoint;
+			Rectangle? lastBounds = view?.Frame;
+
+			nativeView.UpdateTransformation(view, layer, isInteractive, originalAnchor, lastBounds);
+		}
+
+		public static void UpdateTransformation(this UIView nativeView, IView? view, CALayer? layer, bool isInteractive, CGPoint? originalAnchor, Rectangle? lastBounds)
+		{
+			if (view == null)
+				return;
+
+			bool shouldInteract;
+
+			if (view is ILayout layout)
+				shouldInteract = layout.IsEnabled;
+			else
+				shouldInteract = view.IsEnabled;
+
+			if (isInteractive != shouldInteract)
+			{
+				nativeView.UserInteractionEnabled = shouldInteract;
+				isInteractive = shouldInteract;
+			}
+
+			var boundsChanged = lastBounds != view.Frame;
```

Comment:
This should be in a separate mapper as it is unrelated to transforms.

Reply:
Removed.